pyaedt/generic/plot.py

```diff
@@ -194,28 +194,42 @@ def _parse_aedtplt(filepath):
                 nodes_list = l[l.find("(") + 1 : -2].split(",")
                 nodes_list = [float(i.strip()) for i in nodes_list]
             if "ElemSolution(" in l:
-                # convert list of strings to list of floats
-                sols = l[l.find("(") + 1 : -2].split(",")
-                sols = [is_float(value) for value in sols]
-                # sols = [float(i.strip()) for i in sols]
+                solStr = l[l.find("(") + 1 : -2].split(",")
```

Comment:
I would suggest, if possible, to rename the variable `solStr`. It could be more descriptive and follow snake_case Python naming convention.
This should help for future maintenance.